lib/ansible/utils/module_docs_fragments/ldap.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2017-2018 Keller Fuchs (@kellerfuchs) <kellerfuchs@hashbang.sh>
```

Comment:
The right order should be:

```
# Copyright 2017-2018 Keller Fuchs (@kellerfuchs) <kellerfuchs@hashbang.sh>
# Copyright 2016 Jiri Tyr <jiri.tyr@gmail.com>
# Copyright 2016 Peter Sagerson <psagers@ignorare.net>
```

Reply:
This was literally copy-pasted from the original file.

Reply:
The original modules have this:

```yaml
# (c) 2016, Peter Sagerson <psagers@ignorare.net>
# (c) 2016, Jiri Tyr <jiri.tyr@gmail.com>
```

Peter is the original author, I'm the person who polished and published his module. You are another contributor, so you should be at the end of the list.

Reply:
Thanks for clarifying, because it was utterly unclear to me that those lines should be in chronological order, rather than reverse chronological + alphabetical order.

Reply:
Eeeeeh, this is literally the opposite of [what you previously said](https://github.com/ansible/ansible/pull/33040#discussion_r182894470)

Reply:
> You are another contributor, so you should be at the end of the list.

And yet for your comment six days ago, @KellerFuchs was on top?